Make simple mobs take different amounts of damage

PurpleMartinJCK opened the pull request:
- Depends on unarmed attack used
- Uses the unarmed attack's verb
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->
🆑 FTangSteve
rscadd: simple mobs now take different damage from different unarmed attacks
/🆑

PurpleMartinJCK committed:
Make simple mobs take different amounts of damage depending on the unarmed attack used

MistakeNot4892 commented on code/modules/mob/living/carbon/human/unarmed_attack.dm:
```diff
@@ -23,13 +23,19 @@ var/global/list/sparring_attack_cache = list()
 
+/datum/unarmed_attack/proc/get_simple_damage()
+	var/simple_damage = damage
+	simple_damage += damage * edge
```
Why are edge/sharp being used as multipliers? Why are they cumulative?

MistakeNot4892 commented:
It feels quite strange creating a new set of procs and calls for the sake of a mob type rather than just having that mob type call the usual procs with different args.

PurpleMartinJCK committed:
Updates and changes
- Make simple mobs use bleed
- Have bleed use different colours
- Sharp unarmed attacks cause mobs to bleed

PurpleMartinJCK commented:
Updated
